app/src/main/res/layout/language_items.xml

```diff
@@ -27,7 +24,7 @@
       android:layout_height="wrap_content"
       android:layout_marginStart="24dp"
       android:layout_marginEnd="24dp"
-      android:checked="@{isChecked}"
+      android:checked="@{viewModel.isLanguageSelected()}"
```

Comment:
I think you can drop the ``()`` and treat the live data as a value.

Reply:
Done